tests/lifecycle/test_case_test.go

```diff
@@ -19,11 +21,12 @@ import (
 type serviceLifecycleTestCase struct {
 	module                 service.Module
 	description            string
-	setup                  func() error
 	serviceID              string
 	planID                 string
 	location               string
 	provisioningParameters service.ProvisioningParameters
+	parentService          *service.Instance
```

Comment:
Can we change the name of this attribute? At a glance, it looks like a service and not a service instance. If I'm more careful about noticing the type, it becomes clear, but I don't want the name of the attribute itself to mislead anyone.